exercises/wordy/canonical-data.json

```diff
@@ -151,6 +151,14 @@
       },
       "expected": {"error": "syntax error"}
     },
+    {
+      "description": "reject unstarted problem",
```

Comment:
if I call it unfinished, I worry that the description sounds too similar to `incomplete problem` above (since I see `unfinished` and `incomplete` to have too similar meanings). 

If we take for granted that "What is" is a common feature of all questions, then I see it making sense since they didn't even start saying anything else

But if this assumption gets broken in a later PR that adds questions that don't start with "What is", then this description won't make sense!!

Reply:
Perhaps `ambiguous` instead of `unfinished`.

Reply:
I think the way out will be to say exactly in what way the two problems are incomplete. I have now made a proposal.